distributed/shuffle/_shuffle_extension.py

```diff
@@ -252,30 +328,24 @@ async def shuffle_receive(
         Using an unknown ``shuffle_id`` is an error.
         """
         shuffle = await self._get_shuffle(shuffle_id)
-        task = asyncio.create_task(shuffle.receive(data))
-        if (
-            shuffle.multi_file.total_size + sum(map(len, data))
-            > shuffle.multi_file.memory_limit
-        ):
-            await task  # backpressure
-
-    async def shuffle_inputs_done(self, comm: object, shuffle_id: ShuffleId) -> None:
+        await shuffle.receive(data)
+
+    async def shuffle_inputs_done(self, shuffle_id: ShuffleId) -> None:
         """
         Hander: Inform the extension that all input partitions have been handed off to extensions.
```

Comment:
Driveby: 
```suggestion
        Handler: Inform the extension that all input partitions have been handed off to extensions.
```
(same in L327)